Gravity/Gravity.Test.Unit/RsapiDaoGetTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using NUnit.Framework;
```

Comment:
I do not see this load with the project.  I do see it on disk.  Is that intentional?  I think it is fine to add to project.

Reply:
This has been taken care of in the last commit.